extension-iap/src/iap_ios.mm

```diff
@@ -119,8 +119,8 @@ - (void)productsRequest:(SKProductsRequest *)request didReceiveResponse:(SKProdu
 
         IAPProduct product = {0};
         product.ident           = strdup([p.productIdentifier UTF8String]);
-        product.title           = strdup([p.localizedTitle UTF8String]);
-        product.description     = strdup([p.localizedDescription UTF8String]);
+        product.title           = p.localizedTitle ? strdup([p.localizedTitle UTF8String]) : strdup([p.productIdentifier UTF8String]);
```

Comment:
Should we return an empty string or the product id if title doesn't exist?

Reply:
I think, yes. Maybe it makes sense to show warning for a developer?